lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -50,6 +51,8 @@
                     this.discussionView.render();
 
                     this.renderTeamMembers();
+
+                    $('.sr-is-focusable').focus();
```

Comment:
This should be scoped to the view by using `this.$()`. Because there may be other elements in the page with the same class.